src/components/Plan.js

```diff
@@ -54,6 +92,18 @@ class Plan extends Component {
       hackLocationButton: 0
     }),500);
   }
+
+  componentDidMount(){
+    AsyncStorage.getItem(FAV_KEY).then(res =>{
```

Comment:
It is a good practice to create an abstraction on top of 'AsyncStorage'. Maybe  you can try this as reference https://github.com/jasonmerino/react-native-simple-store/blob/master/src/index.js